site/src/sphinx/setup-installation.rst

```diff
@@ -6,13 +6,13 @@ Installation
 Prerequisites
 -------------
 Central Dogma server is a Java application that requires Java 8 or above. If your system has no Java installed,
-you need to `download and install Java <http://www.oracle.com/technetwork/java/javase/downloads/>`_ first.
+you need to `download and install Java <https://www.oracle.com/technetwork/java/javase/downloads/>`_ first.
 Once installed, make sure the ``java`` binary is in your ``PATH`` or the ``JAVA_HOME`` environment variable is
 set correctly.
 
 If you have a plan to use multi-master replication, you also need to set up an
 `Apache ZooKeeper <https://zookeeper.apache.org>`_ cluster.
-Refer to `the ZooKeeper administrator's guide <http://zookeeper.apache.org/doc/r3.4.10/zookeeperAdmin.html>`_
+Refer to `the ZooKeeper administrator's guide <https://zookeeper.apache.org/doc/r3.4.10/zookeeperAdmin.html>`_
```

Comment:
This is not true anymore. Could you remove this entire paragraph taking advantage of this change? :pray: